bindings/cpp/include/wtf/platform.h

```diff
@@ -45,16 +45,23 @@ std::string PlatformGetThreadName();
 #include "wtf/platform/platform_default_inl.h"
 #endif
 
+#if !defined(WTF_SINGLE_THREADED) && !defined(WTF_PTHREAD_THREADED) && !defined(WTF_STD_THREADED)
+#warning "Neither of WTF_SINGLE_THREADED, WTF_PTHREAD_THREADED, WTF_STD_THREADED was defined, defaulting to pthread"
+#define WTF_PTHREAD_THREADED
+#endif
+
 // Select threading library.
 #if defined(WTF_SINGLE_THREADED)
 // Process is single threaded so avoid TLS.
 #include "wtf/platform/platform_aux_single_threaded_inl.h"
 #elif defined(WTF_PTHREAD_THREADED)
 // Explicitly use pthreads instead of std::thread.
 #include "wtf/platform/platform_aux_pthreads_threaded_inl.h"
-#else
-// Default to std::thread and friends.
+#elif defined(WTF_STD_THREADED)
```

Comment:
Per the comment in the Makefile, it is best if this is the default, not requiring a macro.